includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -194,6 +195,7 @@ class AMP_Allowed_Tags_Generated {
 			'hkern',
 			'hr',
 			'i',
+			'img',
```

Comment:
Also apparently not allowed in AMP Stories, see https://github.com/ampproject/amphtml/blob/4ce3cd79520dbeaf5ed5364cbff58d3d71dee40e/extensions/amp-story/validator-amp-story.protoascii#L598-L711